store/aciinfo.go

```diff
@@ -21,46 +21,57 @@ import (
 	"time"
 )
 
+// ACIFetchInfo is used to store information about the fetching process of an ACI.
+type ACIFetchInfo struct {
```

Comment:
I'm ok on keeping an unique db table (aciinfo).
This splitting have a double function: 
* avoid having a big number of arguments in WriteACI
* just reuse this struct when creating the ACIInfo struct inside WriteACI.

On the other side IMHO it's not a good separation since the ACIFetchInfo struct is used for a double reason.

Perhaps removing this struct and add the new variables to ACIInfo will be clear. While for WriteACI just use multiple arguments, or if they will increase, create a dedicated struct that will be used only as a WriteACI argument.